### ISSUE TITLE
When Mesh Viewer does not match Viewer Draw Node

### ISSUE THREAD

**Issue by enzyme69** [nortikin/sverchok#3522]
<img width="877" alt="Screen Shot 2020-09-08 at 10 47 25 am" src="https://user-images.githubusercontent.com/3952687/92422464-e1a62b80-f1c0-11ea-9adb-ad2a58fb5dd2.png">

[render_sverchok_test_001.blend.zip](https://github.com/nortikin/sverchok/files/5185503/render_sverchok_test_001.blend.zip)

I found a glitch, mesh viewer does not output the correct expected result.

**Comment by enzyme69** [comment]
Very likely it's the Subdiv and Subdiv Lite issue.

**Comment by Durman** [comment]
The problem is how I reimplemented **Mesh viewer** node. It is significantly faster only to fix position of points if topology was unchanged neither rebuild all mesh from scratch. For this reason I have added simple test of recognizing whether topology was changed or not. It just tests whether number of vertices, edges and faces was not changed. But definitely such test can't recognize all cases of topology changes and this is unfortunately one of them. 

It is possible to switch off this test on N panel and the node will consider that topology was changed every update event.

![2020-09-08_08-10-06](https://user-images.githubusercontent.com/28003269/92432272-c03a4500-f1aa-11ea-903a-298620da8d38.png)

To make things a little bit clear I'm talking about huge speed increase. It can be 10 times faster. Have a look here:
https://github.com/nortikin/sverchok/pull/3475#issuecomment-680010778

This was the reson why this option is True by default. Probably, it would be possible to move this option to the node UI so users could have more experiments with it.

Also I'll have a look whether it will be possible to improve test for recognizing such cases.
